agent/src/beerocks/slave/backhaul_manager/backhaul_manager_thread.cpp

```diff
@@ -3007,7 +3019,7 @@ bool backhaul_manager::handle_1905_topology_query(ieee1905_1::CmduMessageRx &cmd
     }
 
     size_t number_of_supported_services = 1;
-    if (local_master) {
+    if (db->device_conf.local_controller) {
```

Comment:
```suggestion
    auto local_controller = db->device_conf.local_controller;
    if (local_controller) {
```
for `local_controller` it is even easier

Reply:
Again, I don't think we need to copy values from a struct... Should simply access them as is.